packages/core/core/src/RequestTracker.js

```diff
@@ -145,6 +145,7 @@ export type RunAPI = {|
   getRequestResult<T>(contentKey: ContentKey): Async<?T>,
   getPreviousResult<T>(ifMatch?: string): Async<?T>,
   getSubRequests(): Array<StoredRequest>,
+  getInvalidSubRequests(): Array<StoredRequest>,
```

Comment:
This new API is subtly different than `api.getSubRequest().filter(req => !api.canSkipSubrequest(req.id))` in that it doesn't have the side effect of preserving all previous sub requests even if not used.